js/ui/data_grid/ui.data_grid.virtual_scrolling.js

```diff
@@ -381,7 +381,9 @@ var VirtualScrollingRowsViewExtender = (function() {
                 };
 
             for(i = 0; i < columns.length; i++) {
-                freeSpaceCellsHtml += columns[i].command === "expand" ? "<td class='" + DATAGRID_GROUP_SPACE_CLASS + "'/>" : "<td />";
+                var cssClass = columns[i].cssClass;
+
+                freeSpaceCellsHtml += columns[i].command === "expand" ? "<td class='" + DATAGRID_GROUP_SPACE_CLASS + " " + cssClass + "'/>" : "<td class='" + cssClass + "'/>";
```

Comment:
if cssClass is not assigned, the class attribute for "expand" column will be 'dx-datagrid-group-space ' with space at the and.